docs/index.md

```diff
@@ -24,6 +24,12 @@ To install the Arcus.WebApi.Logging package:
 PM > Install-Package Arcus.WebApi.Logging
 ```
 
```

Comment:
I'm wondering if we actually need to start listing all of these packages or move it to the more specific docs? But that's something for a separate PR I guess

Reply:
This is ok for now, will sweep once this one is merged.